model/map-jpa/src/main/resources/META-INF/auth-events/jpa-auth-events-changelog-1.xml

```diff
@@ -0,0 +1,71 @@
+http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd
+                    http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd">
+
+    <!-- format of id of changeSet: auth-events-${org.keycloak.models.map.storage.jpa.Constants.CURRENT_SCHEMA_VERSION_AUTH_EVENT} -->
+    <changeSet author="keycloak" id="auth-events-1">
+
+        <createTable tableName="kc_auth_event">
+            <column name="id" type="UUID">
+                <constraints primaryKey="true" nullable="false"/>
+            </column>
+            <column name="version" type="INTEGER" defaultValueNumeric="0">
+                <constraints nullable="false"/>
+            </column>
+            <column name="metadata" type="json"/>
+        </createTable>
+        <ext:addGeneratedColumn tableName="kc_auth_event">
+            <ext:column name="entityversion" type="INTEGER" jsonColumn="metadata" jsonProperty="entityVersion"/>
+            <ext:column name="realmid" type="KC_KEY" jsonColumn="metadata" jsonProperty="fRealmId"/>
+            <ext:column name="timestamp" type="BIGINT" jsonColumn="metadata" jsonProperty="fTimestamp"/>
+            <ext:column name="expiration" type="BIGINT" jsonColumn="metadata" jsonProperty="fExpiration"/>
+        </ext:addGeneratedColumn>
+        <createIndex tableName="kc_auth_event" indexName="auth_event_entityversion">
+            <column name="entityversion"/>
+        </createIndex>
+        <createIndex tableName="kc_auth_event" indexName="auth_event_realmid_time">
+            <column name="realmid"/>
+            <column name="timestamp"/>
+        </createIndex>
+
+        <createTable tableName="kc_auth_event_detail">
+            <column name="id" type="UUID">
+                <constraints primaryKey="true" nullable="false"/>
+            </column>
+            <column name="fk_root" type="UUID">
+                <constraints foreignKeyName="auth_event_detail_fk_root_fkey" references="kc_auth_event(id)" deleteCascade="true"/>
+            </column>
+            <column name="name" type="VARCHAR(255)"/>
+            <column name="value" type="TEXT"/>
+        </createTable>
+        <createIndex tableName="kc_auth_event_detail" indexName="auth_event_detail_fk_root">
+            <column name="fk_root"/>
+        </createIndex>
```

Comment:
Question: This is similar to the attributes tables we have for other entities, and those have an index for the name/value. I haven´t added such index here because the details are never queried by name or value, they are always loaded in their entirety, so to me only the fk_root index should be enough

Reply:
+1 for only having `fk_root` index